### PR TITLE
brew_services: use `HOMEBREW_BREW_FILE`

### PR DESCRIPTION
**Pull request by carlocab**
`brew` probably isn't in `PATH` here. While we're here, let's avoid the
backticks.

Fixes Homebrew/brew#13611.

### CODE REVIEW

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Great catch, nice work @carlocab!

**Comment by carlocab** [comment]
No write access, so this is ready to merge when you are, @MikeMcQuaid. (Not sure about the tests though.)

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks @carlocab!

**Pull request merged**
